src/main/java/seedu/address/model/task/exceptions/TaskNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.task.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified task.
+ */
+public class TaskNotFoundException extends RuntimeException {}
```

Comment:
Can I check if you will be adding any error message to this exception class?

Reply:
Just added, will push